addons/vehiclelock/functions/fnc_onOpenInventory.sqf

```diff
@@ -37,6 +37,12 @@ if (GVAR(LockVehicleInventory) && //if setting not enabled
     _handeled = true;
     //Just opens a dummy groundContainer (so the player can still see their own inventory)
     ACE_player action ["Gear", objNull];
+
+    //As of right now arma doesn't seem to be respecting this anymore, delaying a frame seems to work.
```

Comment:
I never noticed it until now, so it could have broken in 1.50, or 1.52.  But this fix will work on 1.54